apollo-compiler/src/main/kotlin/com/apollostack/compiler/OperationTypeSpecBuilder.kt

```diff
@@ -1,26 +1,43 @@
 package com.apollostack.compiler
 
+import com.apollostack.compiler.ir.CodeGenerator
 import com.apollostack.compiler.ir.Field
-import com.cesarferreira.pluralize.singularize
+import com.apollostack.compiler.ir.Fragment
 import com.squareup.javapoet.TypeSpec
 import javax.lang.model.element.Modifier
 
 class OperationTypeSpecBuilder(
```

Comment:
Yeah I thought about doing that. Sounds like it could be a good idea for consistency